client/src/constants/actionTypes.js

```diff
@@ -14,5 +14,10 @@ export default {
   OAUTH_START: 'OAUTH_START',
   OAUTH_DONE: 'OAUTH_DONE',
   AUTHORIZED_USER: 'AUTHORIZED_USER',
-  LOGOUT_USER: 'LOGOUT_USER'
+  LOGOUT_USER: 'LOGOUT_USER',
+  CLEAR_MESSAGE: 'CLEAR_MESSAGE',
+  ADD_MESSAGE: 'ADD_MESSAGE',
```

Comment:
CLEAR_MESSAGE and ADD_MESSAGE can be deleted. If you don't you'll have a conflict on this file when merging

Reply:
Well, maybe I lied. These do need to be deleted though